packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -48,13 +50,9 @@ import {
   updateTextType,
 } from '../utils';
 import style from './style.css';
-import {
-  CaptionIcon,
-  CopyIcon,
-  DeleteIcon,
-  DownloadIcon,
-} from '../../image-block/icons';
+import { CaptionIcon, CopyIcon, DeleteIcon, DownloadIcon } from '../icons';
 import { downloadImage, focusCaption, copyImgToClip } from './utils';
+import { isAtLineEdge } from '../../__internal__/rich-text/rich-text-operations';
```

Comment:
Just saying that we usually add a blank line after all `import` statements. Similarly, a blank line should also be added after we defined the `EmbedOption` interface (well `EmbedEditingState` seems to be a better fix here). It is recommended to refer to the existing code base to determine when crowded code should be avoided.